packages/ember-htmlbars/tests/helpers/input_test.js

```diff
@@ -1,492 +0,0 @@
-import run from 'ember-metal/run_loop';
-import { set } from 'ember-metal/property_set';
-import View from 'ember-views/views/view';
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-import compile from 'ember-template-compiler/system/compile';
-import ComponentLookup from 'ember-views/component_lookup';
-import TextField from 'ember-views/views/text_field';
-import Checkbox from 'ember-views/views/checkbox';
-import EventDispatcher from 'ember-views/system/event_dispatcher';
-import buildOwner from 'container/tests/test-helpers/build-owner';
-import { OWNER } from 'container/owner';
-
-var view;
-var controller, owner;
-
-function commonSetup() {
-  owner = buildOwner();
-  owner.register('component:-text-field', TextField);
-  owner.register('component:-checkbox', Checkbox);
-  owner.register('component-lookup:main', ComponentLookup);
-  owner.register('event_dispatcher:main', EventDispatcher);
-
-  var dispatcher = owner.lookup('event_dispatcher:main');
-  dispatcher.setup({}, '#qunit-fixture');
-}
-
-import isEnabled from 'ember-metal/features';
-if (!isEnabled('ember-glimmer')) {
-  // jscs:disable
-
-QUnit.module('{{input type=\'text\'}}', {
-  setup() {
-    commonSetup();
-
-    controller = {
-      val: 'hello',
-      place: 'Enter some text',
-      name: 'some-name',
-      max: 30,
-      size: 30,
-      tab: 5
-    };
-
-    view = View.extend({
-      [OWNER]: owner,
-      controller,
-      template: compile('{{input type="text" disabled=disabled value=val placeholder=place name=name maxlength=max size=size tabindex=tab}}')
-    }).create();
-
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-  }
-});
-
-QUnit.test('should insert a text field into DOM', function() {
-  equal(view.$('input').length, 1, 'A single text field was inserted');
-});
-
-QUnit.test('should become disabled if the disabled attribute is true', function() {
-  ok(view.$('input').is(':not(:disabled)'), 'There are no disabled text fields');
-
-  run(null, set, controller, 'disabled', true);
-  ok(view.$('input').is(':disabled'), 'The text field is disabled');
-
-  run(null, set, controller, 'disabled', false);
-  ok(view.$('input').is(':not(:disabled)'), 'There are no disabled text fields');
-});
-
-QUnit.test('input value is updated when setting value property of view', function() {
```

Comment:
moved to `@htmlbars Helpers test: {{input}}` : [`@test dynamic attributes`](https://github.com/emberjs/ember.js/pull/13194/files#diff-0caed08a992497ff104284ec6aea6d96R108)